hotline/flatpages/views.py

```diff
@@ -0,0 +1,79 @@
+from arcutils import will_be_deleted_with
+from django.shortcuts import render, get_object_or_404, redirect
+from django.core.exceptions import ObjectDoesNotExist
+from django.contrib import messages
+from django.contrib.sites.models import Site
+from django.contrib.flatpages.models import FlatPage
+
+from hotline.flatpages.forms import FlatterPageForm
+
+from .perms import permissions
+
+
+def _list(request):
```

Comment:
A _ prefix usually means something is "private". A _ suffix means the name without the suffix collides with a Python name. So this should be `list_`